config/sync/core.entity_form_display.node.stanford_page.default.yml

```diff
@@ -78,6 +78,15 @@ content:
   su_page_components:
     type: react_paragraphs
     settings:
+      sizes:
+        stanford_banner: 1
+        stanford_card: 1
+        stanford_wysiwyg: 1
+        stanford_media_caption: 1
+        stanford_spacer: 1
+        stanford_cta_list: 12
+        stanford_image_cta: 1
+        stanford_stories: 12
```

Comment:
If I'm reading this right, it appears these settings would allow up to twelve banners, cards, wysiwygs, spacers, or image_ctas per row.  Is that correct?

Reply:
@imonroe I thought it mean 12 columns. So you'd have 1 image_cta and stanford_stories. I will come back and retest.

Reply:
Okay, I retested it and it works like I'd expect. All paragraphs except the Stories and Image CTA can have 4 per row. Only those are taking the full 12 columns.

Reply:
Sounds good to me.